client/src/panels/panel_declarations/results/result_components.js

```diff
@@ -329,9 +329,8 @@ const StatusIconTable = ({
             >
               {result_simple_statuses[status_key].text}
             </span>
-          ) : (
-            result_simple_statuses[status_key].text
           ),
+          ariaText: result_simple_statuses[status_key].text,
```

Comment:
Minor point, we usually use snake case for props. Not 100% consistent, we tended to go back and forth very early on. Personally, I prefer to reserve camel case for the HTML attribute props (like `className`, etc).